src/main/java/com/michelin/ns4kafka/service/ResourceQuotaService.java

```diff
@@ -54,15 +55,43 @@ public class ResourceQuotaService {
     ConnectorService connectorService;
 
     /**
-     * Find a resource quota by namespace.
+     * Find a resource quota of a given namespace.
      *
      * @param namespace The namespace used to research
      * @return The researched resource quota
      */
-    public Optional<ResourceQuota> findByNamespace(String namespace) {
+    public Optional<ResourceQuota> findForNamespace(String namespace) {
         return resourceQuotaRepository.findForNamespace(namespace);
     }
 
+    /**
+     * Find all the resource quota of a given namespace.
+     *
+     * @param namespace The namespace used to research
+     * @return The researched resource quota
+     */
+    public List<ResourceQuota> findAllForNamespace(String namespace) {
```

Comment:
Literally the same method than `resourceQuotaRepository#findForNamespace`, right ?

```java
getKafkaStore()
  .values()
  .stream()
  .filter(resourceQuota -> resourceQuota.getMetadata().getNamespace().equals(namespace))
```

As it is used only from `resourceQuotaService#findByWildcardName`, you can just do:

```java
findForNamespace(namespace)
  .stream()
  .filter(quota -> RegexUtils.filterByPattern(quota.getMetadata().getName(), nameFilterPatterns))
  .toList();
```

Optional can be converted to stream: https://javadoc.scijava.org/Java13/java.base/java/util/Optional.html#stream()